lib/td/client/api/table.rb

```diff
@@ -81,9 +82,19 @@ def update_schema(db, table, schema_json)
   # @param [Fixnum] expire_days
   # @return [true]
   def update_expire(db, table, expire_days)
-    code, body, res = post("/v3/table/update/#{e db}/#{e table}", {'expire_days'=>expire_days})
+    update_table(db, table, {:expire_days=>expire_days})
+  end
+
+  # @param [String] db
+  # @param [String] table
+  # @option opts [Fixnum] :expire_days days to expire table
+  # @option opts [Boolean] :include_v (true) include v column on Hive
+  # @option opts [Boolean] :detect_schema (true) detect schema on import
+  # @return [true]
+  def update_table(db, table, params)
```

Comment:
Doc comment called `opts` but the arg name is `params`. Maybe the same name is better.